EIPS/eip-7702.md

```diff
@@ -43,6 +43,14 @@ rlp([chain_id, nonce, max_priority_fee_per_gas, max_fee_per_gas, gas_limit, dest
 authorization_list = [[chain_id, address, nonce, y_parity, r, s], ...]
 ```
 
+Transaction is considered invalid if authorization list items can't be decoded as:
+
+* `chain_id` and `nonce`: unsigned 64-bit integer.
```

Comment:
`chain_id` should be a uint256

Reply:
Why should it be uint256?

there are no chain IDs that are bigger than u64: https://chainid.network/

Reply:
Both reth and alloy uses u64 for ChainId

Reply:
It hasn't been specified as a uint64 anywhere. To be fair, an explicit bound hasn't been given afaict. But in these cases we basically always assume it is bound by uint256. Reth /alloy can definitely impl as uint64 if you prefer but I wouldn't put that in the spec.

Reply:
I don't mind limiting the chain id, but I would want that to be a separate proposal that limits the chain id across all instances in the protocol at one time.

Reply:
It seems people responded that they want chain_id U256, so will make a change for that.